### PR TITLE
Fix issue7 pytest fixture

### LINKED ISSUE #7: Use pytest fixture to load the core plugins

### PR DESCRIPTION
**Pull request by pscff**
Fixes #7

### CODE REVIEW

File: {{ cookiecutter.project_slug }}/tests/test_parse.py [hunk 1]
```diff
@@ -5,9 +5,7 @@ def test_json():
     from {{ cookiecutter.package_name }}.strategies.parse import DemoJSONDataParseStrategy
     from oteapi.models.resourceconfig import ResourceConfig
     from oteapi.plugins.plugins import load_plugins
```
Comment (CasperWA) [inline {{ cookiecutter.project_slug }}/tests/test_parse.py:7]:
```suggestion
```

I guess this is not needed any more?

**Review by CasperWA — CHANGES_REQUESTED** [review]
Cheers @pscff.

I think an import can be removed? If not - this can be approved.

**Review by CasperWA — APPROVED** [review]
Cheers @pscff

**Review by CasperWA — CHANGES_REQUESTED** [review]
Please run `pre-commit run --all-files` and commit and push the changed files.

**Review by pscff — COMMENTED** [review]
Requested changes done.

**Review by CasperWA — APPROVED** [review]
Look good. I see you've added a `.gitignore` as well. Couldn't hurt 👍

**Pull request merged**